### PR TITLE
Transfer vanity to another underlying url

### PR DESCRIPTION
**Pull request by ookejimift**

### CODE REVIEW

File: client/components/underlyManagement.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+			<button className="ums-submit ums-grouped-button"
+					onClick={ toggleVanityOverwrite }>
+				Cancel
+			</button>
+		</div>
+	);
+
+	const overwriteButton = (
+		<button className="ums-submit"
+				onClick={ toggleVanityOverwrite }>
```
Comment (TheoLeanse) [inline client/components/underlyManagement.js]: do we want it to just open up a single input to only edit either the underlying or the vanity?

**Comment by ookejimift** [comment]
@TheoLeanse updated, but I think we might need a `steal` api endpoint

**Comment by TheoLeanse** [comment]
yes I think a steal endpoint makes sense

👍

File: client/components/underlyManagement.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ export default ({ vanity, toURL, create, toggleUnderlyOverwrite, underlyOverwrit
 			<button className="ums-submit ums-grouped-button"
 					onClick={
 					() => {
-						create(vanity.get('fromURL'), newtoURL.value, 100, 'insert_or_update');
+						create(vanity.get('fromURL'), newtoURL.value, 100, 'insert_or_steal');
```
Comment (matthew-andrews) [inline client/components/underlyManagement.js]: steal! haha

**Pull request merged**